### ISSUE TITLE
SHA miner not showing up in WebUI

### ISSUE THREAD

**Issue by DocGonzzo** [HASHRA/CONTROLA#1]
I'm having an issue where one of my Gridseeds does not show up in the WebUI (Dual Mode) as hashing, although it would appear to be working my pool hashrate report. I am only running two units at the moment, so confirming that it is actually hashing is easy. I was attempting to gain root access to the controller, but could not. Would you mind sharing the login info?

**Comment by turnkit** [comment]
I don't know it but if it's the same as the Scripta, I think I've seen that
one posted online.

Best,

On Wed, Mar 26, 2014 at 7:16 PM, richmondimage notifications@github.comwrote:

> I'm having an issue where one of my Gridseeds does not show up in the
> WebUI (Dual Mode) as hashing, although it would appear to be working my
> pool hashrate report. I am only running two units at the moment, so
> confirming that it is actually hashing is easy. I was attempting to gain
> root access to the controller, but could not. Would you mind sharing the
> login info?
> 
> ## 
> 
> Reply to this email directly or view it on GitHubhttps://github.com/HASHRA/CONTROLA/issues/1
> .

## 

David C. Sutherland   |  (310) 729-6411 (c)

**Comment by DocGonzzo** [comment]
Yeah I know how to fix it on Scripta because they provide the login info, I cant login to the hashra, unless I am missing something mind numbingly obvious

**Comment by maxzillabong** [comment]
Hi Richmond,
It's strange you're having this problem. Are you using a powered hub?
I'm sorry to say I don't have the login info on the hashra image. I gave
them one and they changed the password. On bitcointalk someguy posted the
original 4GB image (the one I gave hashra)  online.

good luck!

**Comment by DocGonzzo** [comment]
If I start with the original firmware and then apply the branch update, and set my own root access, do you think I should be able to do an update from the git source?

**Comment by maxzillabong** [comment]
yes, the original already has the branch update.
